lib/components/atoms/Video/Video.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+/**
+ *
+ * Video
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+
+import styles from './Video.style';
+import type { Props } from './types';
+
+const Video = ({
+  width,
+  height,
+  autoPlay,
+  controls,
+  loop,
+  muted,
+  poster,
+  src,
+  type,
+  children,
+  className,
+}: Props): Node => {
+  return (
+    <video
+      width={width}
+      height={height}
+      controls={controls}
+      autoPlay={autoPlay}
+      loop={loop}
+      muted={muted}
+      poster={poster}
+      className={className}
+    >
+      <source src={src} type={type} />
+      {children}
```

Comment:
Whats there for Children?

Reply:
So that we can have any of children here like video's caption